### PR TITLE
Add documentation for MVC architecture of Privly Applications

### PR DESCRIPTION
**Pull request by SummerWish**

### CODE REVIEW

**Comment by SummerWish** [comment]
ps, I changed my editor configuration to automatically remove trailing whitespace so there are some extra modifications in [this commit](https://github.com/SummerWish/privly.github.io/commit/f49a93108a1d9adab22faafbc61cd6c7864d20df).

**Comment by SummerWish** [comment]
the removal of trailing whitespace has been reverted, please let me know if I can do more for this documentation :-)

File: pages/develop.md [hunk 1]
```diff
@@ -122,36 +122,36 @@ and improving the code based on the recommendations found
 [on code climate](https://codeclimate.com/github/privly/privly-applications).
 Then open a pull request! Don't be afraid to submit a small pull request.
 
-[**Level 1**](https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Aprivly+label%3A%22Level+1%22): **Run the Privly-Applications build system.** _([Build System](#BuildSystem))_  
+[**Level 1**](https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Aprivly+label%3A%22Level+1%22): **Run the Privly-Applications build system.** _([Build System](#BuildSystem))_
```
Comment (smcgregor) [inline pages/develop.md]: I think the whitespace here needs to stay here?

**Comment by smcgregor** [comment]
I am going to merge this to master, view it as GitHub renders it, then make whitespace changes as necessary.

**Pull request merged**